cpp/roaring64map.hh

```diff
@@ -844,6 +840,8 @@ class Roaring64Map {
             });
     }
 
+    // TODO: implement `rangeUint64Array`
```

Comment:
```suggestion
```

Same comment here, let us not add too many todos in the code when we have the option of creating GH issues.